migration_toolkit_for_containers/installing-mtc-restricted.adoc

```diff
@@ -24,13 +24,22 @@ include::modules/migration-installing-mtc-on-ocp-4.adoc[leveloffset=+1]
 [id="configuring-replication-repository_{context}"]
 == Configuring a replication repository
 
-You must configure an object storage to use as a replication repository. The {mtc-full} ({mtc-short}) copies data from the source cluster to the replication repository, and then from the replication repository to the target cluster. Multi-Cloud Object Gateway (MCG) is the only supported option for a restricted network environment.
+The Multicloud Object Gateway is the only supported option for a restricted network environment.
 
 {mtc-short} supports the xref:../migration_toolkit_for_containers/about-mtc.adoc#migration-understanding-data-copy-methods_about-mtc[file system and snapshot data copy methods] for migrating data from the source cluster to the target cluster. You can select a method that is suited for your environment and is supported by your storage provider.
 
 All clusters must have uninterrupted network access to the replication repository.
 
 If you use a proxy server with an internally hosted replication repository, you must ensure that the proxy allows access to the replication repository.
 
+[id="{context}_configuring-replication-repository-additional-resources"]
+[discrete]
+=== Additional resources for configuring a replication repository
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.7/html-single/planning_your_deployment/index#disconnected-environment_rhocs[Disconnected environment] in _Red Hat OpenShift Container Storage: Planning your deployment_.
```

Comment:
Same comment about the book title

Reply:
Done. Also updating version to 4.8 now that OCS has released that version.